src/back/services/update.tsx

```diff
@@ -0,0 +1,58 @@
+import logger from 'decentraland-gatsby/dist/entities/Development/logger'
+
+import { Discourse, DiscoursePost } from '../../clients/Discourse'
+import { ProposalAttributes } from '../../entities/Proposal/types'
+import UpdateModel from '../../entities/Updates/model'
+import { UpdateAttributes } from '../../entities/Updates/types'
+import { getUpdateUrl } from '../../entities/Updates/utils'
+import { inBackground } from '../../helpers'
+
+export class UpdateService {
+  static async getById(id: UpdateAttributes['id']) {
+    return await UpdateModel.findOne<UpdateAttributes>({ id })
+  }
+
+  static async getAllByProposalId(proposal_id: ProposalAttributes['id']) {
+    return await UpdateModel.find<UpdateAttributes>({ proposal_id })
+  }
+
+  static async updateWithDiscoursePost(id: UpdateAttributes['id'], discoursePost: DiscoursePost) {
+    return await UpdateModel.update(
+      { discourse_topic_id: discoursePost.topic_id, discourse_topic_slug: discoursePost.topic_slug },
+      { id }
+    )
+  }
+
+  static commentUpdateEditInDiscourse(update: UpdateAttributes) {
+    inBackground(async () => {
+      if (!update.discourse_topic_id) {
+        logger.error('No discourse topic associated to this update', { id: update.id })
+        return
+      }
+
+      await Discourse.get().commentOnPost({
+        topic_id: update.discourse_topic_id,
+        raw: `This project update has been edited by the author. Please check the latest status on the [Governance dApp](${getUpdateUrl(
```

Comment:
Every time an author edits the same update, is this comment posted on the forum?

Reply:
Yes!